d2bs/kolbot/libs/manualplay/MapMode.js

```diff
@@ -44,17 +44,6 @@ const MapMode = {
     Config.SkipLogging = []; // Custom log skip list. Set as three digit item code or classid. Example: ["tes", "ceh", 656, 657] will ignore logging of essences.
     Config.ShowCubingInfo = true; // Show cubing messages on console
 
-    // Gambling config
```

Comment:
The point of having this in the generalSettings is to allow setting it for all characters but still leaving the option to override this in each specific config. It should be added back.

Reply:
understood. reverted.
since gambling is also kind of losing control of the character, i don't override settings anymore.